contents/handbook/small-teams/replay/index.mdx

```diff
@@ -16,22 +16,35 @@ We build the [heads-up display](https://en.wikipedia.org/wiki/Head-up_display) w
 
 ## Q4 2023 Goals
 
-### 🎯 Goal 1 : Project "NoteForce Exploration 3000" (Ben, David)
+### 🎯 Goal 1: 10 Happy Android Replay Users 
 
-Why? Pairing up with Michael, Thomas, and Cory from Team Product Analytics to focus on not just releasing Notebooks, but integrating them seamlessly with PostHog 3000 to make it a great experience to explore any and all data across PostHog.
+We'll also release an alpha of iOS Mobile Replay.
 
-### 🎯 Goal 2 : Mobile Session Replay (Manoel, Paul)
+Why? 
 
-Why? - The people want it and we have the right skills in house to do a good job of this. Replay is the main focus, but making mobile a first-class citizen of PostHog is the real goal.
+We've done the work's base and now we want to release the new Android SDK with session replay and onboard customers to validate the product. 
 
```

Comment:
Didn't we with Tim make a list of the sub-goals/features here? Why don't we mention them on the team page as well? Otherwise especially Goal 2 doesn't say much

Reply:
We did... they're all listed in the mega issue.

Is it enough to link to the mega issue?

Feel free to add suggestions here! Or edit within the PR

Reply:
So that we don't duplicate a lot of content, linking to the mega issue probably is good enough. That one is quite detailed 👍